docs/source/tools/mythxvsc/index.rst

```diff
@@ -0,0 +1,61 @@
+.. meta::
+   :description: MythXvsc is an extension for running MythX smart contract analysis from your favourite IDE.
+
+.. _tools.mythxvsc:
+
+MythXvsc
+============================
+
+MythXvsc is an extension for running MythX smart contract analysis from your favourite IDE.
+
+The extensions provides:
+
+- Smart contract compilation (via solidity VSCode extension)
+- Log in to MythX platform
+- AST extraction from compiled source
+- Submission of analysis
+- Displaying analysis result in VSCode in a linting fashion
+
+
+
+
+Dependencies
+-----
+
+The MythXvsc extension depends on Juan Blanco’s solidity extension_. Make sure to download this from Visual Studio Code Marketplace before installing MythXvsc.
+
+.. _extension: https://marketplace.visualstudio.com/items?itemName=JuanBlanco.solidity
+
+Installation
+-----
+
+Install from the Visual Studio marketplace here_.
+
+.. _here: https://marketplace.visualstudio.com/items?itemName=mirkogarozzo.mythxvsc
+
+Setup
+-----
+After installing the extension please enter your registered MythX ethAddress and password in VSCode user settings. These fields are properties of MythXvsc as shown in the screenshots below:
+
+.. image:: installation.png
+.. image:: user_settings.png
+
+**Please note that the credentials stored this way are exposed to VSCode. Be sure to understand the security risk this entails or contact the extension developers if you don't. We are working on a more robust log-in implementation via a single-token.**
```

Comment:
You can use `.. note::` for this, and it will come out in a box.

In this case, you could instead use `.. warning::`, since it's a big deal.